lib/client/db/dbcmd/dbcmd.go

```diff
@@ -124,15 +124,15 @@ type CLICommandBuilder struct {
 }
 
 func NewCmdBuilder(tc *client.TeleportClient, profile *client.ProfileStatus,
-	db *tlsca.RouteToDatabase, rootClusterName string, opts ...ConnectCommandFunc,
+	db tlsca.RouteToDatabase, rootClusterName string, opts ...ConnectCommandFunc,
```

Comment:
What's the reason behind making those functions accept `tlsca.RouteToDatabase` by value rather than by reference?

Reply:
The struct itself is small, being composed of 4 strings which are basically just opaque pointers. I wanted to make it clear that the route is not to be modified by these functions